target/linux/realtek/image/rtl838x.mk

```diff
@@ -33,6 +33,7 @@ TARGET_DEVICES += d-link_dgs-1210-16
 define Device/d-link_dgs-1210-28
   $(Device/d-link_dgs-1210)
   DEVICE_MODEL := DGS-1210-28
+  DEVICE_PACKAGES += kmod-input-gpio-keys-polled
```

Comment:
The same feature is provided by **kmod-gpio-button-hotplug** selected by [```realtek/Makefile```](https://github.com/openwrt/openwrt/blob/1521d5f453683ca1aae87d025d2892901369ce0c/target/linux/realtek/Makefile#L20), so this line can be dropped.

Reply:
Thanks! Updated

Reply:
Sorry, my bad. Done.